app/uk/gov/hmrc/servicedependencies/DataReloadScheduler.scala

```diff
@@ -16,49 +16,33 @@
 
 package uk.gov.hmrc.servicedependencies
 
-import com.kenshoo.play.metrics.MetricsFilter
-import com.typesafe.config.Config
-import net.ceedubs.ficus.Ficus._
+import com.google.inject.Singleton
 import org.slf4j.MDC
 import play.api._
 import play.api.inject.ApplicationLifecycle
-import play.api.mvc._
-import uk.gov.hmrc.play.config.{ControllerConfig, RunMode}
-import uk.gov.hmrc.play.microservice.filters._
-import uk.gov.hmrc.play.graphite.GraphiteConfig
-import uk.gov.hmrc.play.microservice.bootstrap.JsonErrorHandling
-import uk.gov.hmrc.play.microservice.bootstrap.Routing.RemovingOfTrailingSlashes
-import uk.gov.hmrc.servicedependencies.service.UpdateScheduler
 import play.api.libs.concurrent.Execution.Implicits.defaultContext
-
+import uk.gov.hmrc.servicedependencies.service.UpdateScheduler
 
 import scala.concurrent.Future
-import uk.gov.hmrc.play.microservice.filters.{ LoggingFilter, MicroserviceFilterSupport }
-
-object ControllerConfiguration extends ControllerConfig {
-  lazy val controllerConfigs = Play.current.configuration.underlying.as[Config]("controllers")
-}
-
-object MicroserviceLoggingFilter extends LoggingFilter with MicroserviceFilterSupport {
-  override def controllerNeedsLogging(controllerName: String) = ControllerConfiguration.paramsForController(controllerName).needsLogging
-}
 
-object MicroserviceGlobal extends GlobalSettings
-  with GraphiteConfig with RemovingOfTrailingSlashes with JsonErrorHandling with RunMode with MicroserviceFilterSupport {
+import javax.inject.Inject
 
+@Singleton
+class DataReloadScheduler @Inject()(app: Application, configuration: Configuration, updateScheduler: UpdateScheduler) {
   lazy val appName = "service-dependencies"
-  lazy val loggerDateFormat: Option[String] = Play.current.configuration.getString("logger.json.dateformat")
+  lazy val loggerDateFormat: Option[String] = configuration.getString("logger.json.dateformat")
 
   val repositoryDependenciesReloadIntervalKey = "dependency.reload.intervalminutes"
   val libraryReloadIntervalKey = "library.reload.intervalminutes"
   val sbtPluginReloadIntervalKey = "sbtPlugin.reload.intervalminutes"
 
+  onStart()
```

Comment:
You can remove 'onStart' and put the code to the root of the class

Reply:
done